elm-app/src/elm/Event/Test.elm

```diff
@@ -60,7 +60,25 @@ viewEventTest =
                 viewEvent baseUrl English event1
                     |> Query.fromHtml
                     |> Query.find [ Selector.class "event-date" ]
-                    |> Query.has [ text "" ]
+                    |> Query.children [ tag "span" ]
+                    |> Query.first
+                    |> Query.has [ text "Thu, 01/01/1970 02:00" ]
+        , test "Event with the same Date and different Time" <|
```

Comment:
Do you know how can I ignore the timezone difference ? the tests are failing on travis and passing on my machine because the shown time has 2 hours difference.